### PR TITLE
ES-2779: Will send all namespaces to Backstage

### PR DESCRIPTION
**Pull request by Gaardsholt**
Same functionality as before, but will now send all namespaces with the annotation `downscaler/uptime` to Backstage

### CODE REVIEW

File: .github/workflows/release-chart.yml [hunk 1]
```diff
@@ -70,15 +70,15 @@ jobs:
           ./cr package charts/kcale
 
           # Old upload but doesn't work because of release drafter. 'cr upload' create an release.
-          # ./cr upload -o orkarstoft -r kscale -p .cr-release-packages --release-name-template ${{ steps.get_chart_release.outputs.tag_name }} --token ${{ secrets.GITHUB_TOKEN }}
-        
+          # ./cr upload -o orkarstoft -r nightscaler -p .cr-release-packages --release-name-template ${{ steps.get_chart_release.outputs.tag_name }} --token ${{ secrets.GITHUB_TOKEN }}
```
Comment (OrKarstoft) [inline .github/workflows/release-chart.yml:73]:
```suggestion
          # ./cr upload -o BESTSELLER -r nightscaler -p .cr-release-packages --release-name-template ${{ steps.get_chart_release.outputs.tag_name }} --token ${{ secrets.GITHUB_TOKEN }}
```

File: .github/workflows/release-chart.yml [hunk 2]
```diff
@@ -70,15 +70,15 @@ jobs:
...
 
           # Upload the chart to the release
-          gh release upload ${{ steps.get_chart_release.outputs.tag_name }} ".cr-release-packages/kscale-$CHART_SEMVER.tgz"
+          gh release upload ${{ steps.get_chart_release.outputs.tag_name }} ".cr-release-packages/nightscaler-$CHART_SEMVER.tgz"
       - name: Update Chart index
         run: |
           mkdir -p .cr-index
-          ./cr index -o orkarstoft -r kscale --push --release-name-template ${{ steps.get_chart_release.outputs.tag_name }} --token ${{ secrets.GITHUB_TOKEN }} 
\ No newline at end of file
+          ./cr index -o orkarstoft -r nightscaler --push --release-name-template ${{ steps.get_chart_release.outputs.tag_name }} --token ${{ secrets.GITHUB_TOKEN }}
```
Comment (OrKarstoft) [inline .github/workflows/release-chart.yml:84]:
```suggestion
          ./cr index -o BESTSELLER -r nightscaler --push --release-name-template ${{ steps.get_chart_release.outputs.tag_name }} --token ${{ secrets.GITHUB_TOKEN }}
```

**Review by OrKarstoft — CHANGES_REQUESTED** [review]
Shouldn't these be BESTSELLER?

**Review by OrKarstoft — APPROVED** [review]

**Review by ChristianGroentved — APPROVED** [review]
LGTM

**Pull request merged**